flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -195,9 +196,23 @@ public FlowCheckResponse getFlowState(String flowId) {
         flowCheckResponse.setFlowId(flowId);
         flowCheckResponse.setHasActiveFlow(!completed("Flow", flowId, List.of(), allByFlowIdOrderByCreatedDesc));
         flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(allByFlowIdOrderByCreatedDesc, failHandledEvents));
+        setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, allByFlowIdOrderByCreatedDesc, "flow", flowId);
```

Comment:
similarly to flow chain, maybe move this "flow" to constant

Reply:
@lacikaaa I added two constants in `FlowConstants ` and updated several places in FlowService.java to use constant. Please let me know if it's good!